Dockerfile

```diff
@@ -1,6 +1,6 @@
 ## Base image with python and entrypoint scripts ##
 ## ============================================= ##
-FROM python:3.6.8-alpine3.9 AS base
+FROM python:3.6.9-alpine3.10 AS base
```

Comment:
Is the Alpine update necessary? I think Lucas and I previously decided that it was a good idea to stay ~1 minor version behind the bleeding edge Alpine version. I'd probably prefer to stay on 3.9 until 3.11 is released unless we have a good reason to make the switch.